crypto/x509/x509_local.h

```diff
@@ -75,7 +75,7 @@ struct x509_lookup_method_st {
     int (*init) (X509_LOOKUP *ctx);
     int (*shutdown) (X509_LOOKUP *ctx);
     int (*ctrl) (X509_LOOKUP *ctx, int cmd, const char *argc, long argl,
-                 char **ret);
+                 char **ret, OPENSSL_CTX *libctx, const char *propq);
```

Comment:
As with get_by_subject, you also need to retain this one for backwards compat and introduce a ctrl_with_libctx

Reply:
Yep - silly me..